tests/h/activity/query_test.py

```diff
@@ -143,6 +153,18 @@ def test_does_not_redirect_to_user_page_if_user_does_not_exist(self,
 
         assert check_url(pyramid_request, query) is None
 
+    def test_does_not_remove_user_term_from_query_if_group_does_not_exist(self,
```

Comment:
Typo. This test name should end with `if_user_does_not_exist` rather than `if_group_does_not_exist`.

Reply:
I've pushed a fix for this.